src/app/services/beard/page.tsx

```diff
@@ -0,0 +1,146 @@
+'use client';
+
+import Image from 'next/image';
+import { Header } from '../../../components/Header';
+
+export default function Beard() {
+  const defaultTextClassName = 'leading-8 my-5 text-lg';
+  const defaultImageClassName = 'my-8 max-h-[32rem] object-cover';
+  return (
+    <main>
+      <div
+        className="
+        bg-black
+        py-8
+        bg-[url('/images/beard/beard-bg.jpg')]
+        bg-no-repeat
+        bg-cover
+        bg-[position:center_bottom,_left_bottom_0.3rem]
+        after:bg-[url('/images/bg/white-grunge.svg')]
+        after:bg-repeat-x
+        after:w-full
+        after:h-20
+        after:absolute
+        after:bg-[length:400px]
+        after:bottom-[10]
+        
```

Comment:
Perhaps, this is only my problem, but this part is not displayed on the page
<img width="429" alt="Screenshot 2023-09-13 at 13 12 52" src="https://github.com/Alex-2kZharkov/oldboy-barbershop/assets/119830208/d4e3f726-0798-4698-a719-72edf83df16b">